daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SValue.scala

```diff
@@ -90,8 +90,8 @@ sealed trait SValue {
         SRecord(tycon, fields, mapArrayList(values, v => v.mapContractId(f)))
       case SStruct(fields, values) =>
         SStruct(fields, mapArrayList(values, v => v.mapContractId(f)))
-      case SVariant(tycon, variant, rank, value) =>
-        SVariant(tycon, variant, rank, value.mapContractId(f))
+      case SVariant(tycon, rank, variant, value) =>
```

Comment:
I’m confused, here rank is before variant and a few lines above it’s the other way around? One of those must be off